src/NumericalAlgorithms/Interpolation/PredictedZeroCrossing.hpp

```diff
@@ -16,4 +18,15 @@ namespace intrp {
 double predicted_zero_crossing_value(const std::vector<double>& x_values,
                                      const std::vector<double>& y_values);
 
+/*!
+ * \brief Predicts the zero crossing of multiple functions contained in a vector
+ * of datavectors.
+ *
+ * Fits a set of N linear functions where N is the size of y_values that use the
+ * same x_values for the fit. Each fit determines the zero-crossing
+ * corresponding to that y_value.
+ */
+std::vector<double> predicted_zero_crossing_value(
```

Comment:
No necessarily a change request, but are we sure we wan this to return `std::vector<double>` and not `DataVector`?